app/src/main/java/com/example/cm/data/repositories/UserRepository.java

```diff
@@ -344,15 +371,32 @@ public void getStaticFriends(UserListener<List<User>> listener) {
             return mutableUsers;
         }
 
-        userCollection.whereIn(FieldPath.documentId(), userIds).addSnapshotListener(executorService, (value, error) -> {
-            if (error != null) {
-                return;
-            }
-            if (value != null && !value.isEmpty()) {
-                List<MutableLiveData<User>> users = snapshotToMutableUserList(value);
-                mutableUsers.postValue(users);
+        if (userIds.size() > 10) {
```

Comment:
Make use  of the "List<List<String>> subLists = Lists.partition(userIds, 10);" method and iterate through the subList instead to fix the "+ 10 users" issue?